packages/header/src/Header.stories.tsx

```diff
@@ -49,7 +49,8 @@ BodyAltBackground.args = {
 
 export const Small = Template.bind({});
 Small.args = {
-	...defaultArgs,
+	heading: 'Internal application',
+	subline: undefined,
```

Comment:
```suggestion
```

Reply:
Default args contains logo now, so we still need to use it and set subline to undefined